### ISSUE TITLE
Add ability to remove Art Location listing

### ISSUE THREAD

**Issue by bdruth** [flexion/msab-arts-locator#173]
As an Art Location curator, in order to prevent inaccurate listings from being shown in search results, I want to be able to unpublish an Art Location.

**Acceptance Criteria**:
- [x] Art Location curator is able to remove a listing from appearing in search results after it has been published.

**Tasks**:
- [x] Implement changes
- [x] Test
- [x] Review
- [x] User acceptance

**Comment by bdruth** [comment]
@xpmuyanja - in researching this enhancement, I found that we have existing functionality (that was slightly broken) allowing an Art Location entry to be "unpublished" - basically, once I fixed the "approval" checkbox populating, based on if an entry is currently published or not, I can uncheck the approval, save, and this will remove the entry from search results as well as email the contact email that their entry is not approved.

<img width="199" alt="image" src="https://user-images.githubusercontent.com/444571/122974178-a5127d80-d357-11eb-9070-98584a8a4ed5.png">

to

<img width="364" alt="image" src="https://user-images.githubusercontent.com/444571/122974262-bbb8d480-d357-11eb-8f7c-a101af9454ad.png">

Is this sufficient to satisfy this need? It will require a small fix as the `Approved for publish` wasn't populating with the correct status, so there was no way to "undo" it, but otherwise, it's already working in this way.

**Comment by xpmuyanja** [comment]
Tested the removal of an already published Art location and record for "XP Muyanja015" - City: Cottage Grove was removed from display. However, the email confirming the removal was not received.

**Comment by bdruth** [comment]
> Tested the removal of an already published Art location and record for "XP Muyanja015" - City: Cottage Grove was removed from display. However, the email confirming the removal was not received.

@xpmuyanja - which email were you expecting? User email or admin email?

**Comment by xpmuyanja** [comment]
Resolved:
I retested the "Removal" of already published Art and successfully managed to remove and Art record from site (XP Muyanja019). I also got an email notifying me that my published Art record has been removed

![Remover Art location 6-29-21](https://user-images.githubusercontent.com/86261039/123807942-eb1a9480-d8b5-11eb-8a88-d8a18bbbede5.png)

"see email body to confirm removal of my Art record"
Thank you for your interest in being listed on Arts Around MN.
The Minnesota State Arts Board reserves the right to approve or deny any content submitted. Based on our review of the information you submitted, we have determined that we won’t be able to add your listing to the site. 
Sincerely,
The Minnesota State Arts Board
